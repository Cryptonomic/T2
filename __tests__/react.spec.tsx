import React from 'react';
import { shallow } from 'enzyme';

import NodeStatus from '../src/components/NodesStatus';

describe('component', () => {
    it("renders correctly", () => {
        const tree = shallow(<NodeStatus message="aaa" />);
        expect(tree).toMatchSnapshot();
      });
  });
