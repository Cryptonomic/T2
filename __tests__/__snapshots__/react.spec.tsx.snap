// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component renders correctly 1`] = `ShallowWrapper {}`;
